internal/fptest/networkinstance.go

```diff
@@ -46,7 +46,7 @@ func AssignToNetworkInstance(t testing.TB, d *ondatra.DUTDevice, i string, ni st
 	}
 }
 
-// EnableGribiOnNetworkInstance enables Gribi protocol under network instance.
+// EnableGribiUnderNetworkInstance enables Gribi protocol under network instance.
 func EnableGribiUnderNetworkInstance(t testing.TB, d *ondatra.DUTDevice, ni string) {
```

Comment:
Can you help file another PR to rename this func name? https://google.github.io/styleguide/go/decisions#initialisms

It should be `EnableGRIBIUnderNetworkInstance`

Reply:
Sure. Filing PR to rename this function name as mentioned.